lib/file-store.js

```diff
@@ -63,3 +66,27 @@ module.exports.delete = async (file_id, authn_user_id) => {
     await sqldb.callAsync('files_delete', params);
     debug('delete(): soft-deleted row in DB');
 };
+
+/**
+ * Get a file from the file store.
+ *
+ * @param {number} file_id - The file to get.
```

Comment:
Specify that the return value is a `Buffer` (or an object containing a Buffer and a file object)?

Reply:
done